x/ccv/provider/keeper/hooks_test.go

```diff
@@ -53,14 +53,14 @@ func TestValidatorConsensusKeyInUse(t *testing.T) {
 
 		gomock.InOrder(
 			mocks.MockStakingKeeper.EXPECT().GetValidator(ctx,
-				newValidator.SDKStakingOperator(),
+				newValidator.SDKStakingValidator().GetOperator(),
```

Comment:
Does this make the line more understandable?
```suggestion
				newValidator.OperatorSDKPubKey(),
```

Reply:
`GetValidator` takes a validator op address, not a pubkey. But this line can definitely be simplified, thanks. 
```suggestion
				newValidator.SDKValOpAddress()
,
```